reana_commons/k8s/api_client.py

```diff
@@ -42,27 +42,27 @@ def create_api_client(api="BatchV1"):
     k8s_config.load_incluster_config()
     api_configuration = client.Configuration()
     api_configuration.verify_ssl = False
-    if api == "extensions/v1beta1":
-        api_client = client.ExtensionsV1beta1Api()
-    elif api == "CoreV1":
+    if api == "CoreV1":
         api_client = client.CoreV1Api()
     elif api == "StorageV1":
         api_client = client.StorageV1Api()
     elif api == "AppsV1":
         api_client = client.AppsV1Api()
-    elif api == "networking.k8s.io/v1beta1":
-        api_client = client.NetworkingV1beta1Api()
+    elif api == "networking.k8s.io/v1":
+        api_client = client.NetworkingV1Api()
```

Comment:
Please write a line in `CHANGES.rst` about this. We shall target `0.9.0a1` release.

Reply:
Yes, that's good. You can also use `reana-dev git-create-release-commit -c .` to create an explicit release commit.

BTW this will need to amend cluster components's `setup.py` requirements to bump minimal requirements. So a set of PR to everything on the client and cluster side will be necessary. I can pair up on this in the afternoon.

Reply:
Yes, let's pair up. Not sure how to do release and `reana-dev git-create-release-commit -c .` command is not working (need to tag commit).

Reply:
Found a problem with the `release-commit` command. Add a release commit.